avocado/core/job.py

```diff
@@ -33,6 +33,7 @@
                loader, nrunner, output, result, tags, varianter, version)
 from .future.settings import settings
 from .output import LOG_JOB, LOG_UI, STD_OUTPUT
+from .resolver import (resolve, ReferenceResolutionResult)
```

Comment:
Do you suggest we switch to this import style? I mean, as it stands, it's confusing for people trying to contribute if they mean to follow a given style.

Reply:
Looking one more time to it, I agree with @clebergnu; we should define a standard and follow it.

Reply:
According to PEP8 is ok to import like that, this was also a result of a `isort` command that automatically replaced this. I'm ok with both approaches here.

Reply:
I'm also okay with any of the approaches, but I think we should define and follow one of them as our standard. It is just a matter of making a choice here as the style we will follow.

Reply:
Both styles are accepted by PEP8 and will not break any linter. I prefer this way because:

1. `isort` does the job automatically and;
2. reduces the number of lines

Reply:
Since that the comment was here and not on the previous commit, I'm assuming that maybe the () might be strange here. And it was added by mistake. So, is this better?

`from .resolver import ReferenceResolutionResult,  resolve`

I also did this to reduce the length of the lines at the code when using this and also to avoid import the entire module when we are using only one method and one enum.

Reply:
I went true the PEP8 about _imports_ and I like the approach:

`from package import foo, bar, foz, baz`

I'm in about using this style from now on. But again, I think we should decide to go with one or another, even if PEP8 mention both are fine.